crypto/evp/evp_fetch.c

```diff
@@ -250,7 +250,7 @@ static void destruct_evp_method(void *method, void *data)
 static void *
 inner_evp_generic_fetch(struct evp_method_data_st *methdata,
                         OSSL_PROVIDER *prov, int operation_id,
-                        const char *name, const char *properties,
+                        const char *name, ossl_unused const char *properties,
```

Comment:
were you getting a warning here for the FIPS_MODULE case? I assume this is just a hint to not issue a warning, but its a bit misleading in the non FIPS case..

Reply:
It was to avoid a warning.
It is just a hint so shouldn't be harmful.

Happy to remove it so long as we accept that there will be complaints.  I'm not so keen on making it conditional using the preprocessor -- that would be ugly.